core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2657,6 +2657,24 @@ private[spark] object Utils extends Logging {
     try f.apply(resource) finally resource.close()
   }
 
+  /**
+   * Closes the given Closeable as a null-safe operation while consuming IOException
+   * by the given consumer.
+   *
+   * @param closeable The resource to close, may be null.
+   * @param consumer Consumes the IOException thrown by Closeable#close().
+   */
+  def closeQuietly(closeable: Closeable, consumer: IOException => Unit = null): Unit = {
```

Comment:
Is the logic identical, @LuciferYang ?

Reply:
The logic is the same, but `IOUtils.closeQuietly ` has been identified as `@Deprecated `.

And I found a similar method encapsulation in `o.a.s.network.util.JavaUtils.java`, should we reuse it?

https://github.com/apache/spark/blob/c040593fb432f75f7e22e3d32a6979142a125375/common/network-common/src/main/java/org/apache/spark/network/util/JavaUtils.java#L50-L58

Reply:
Thank you for the confirmation. Yes, we had better reuse it.

Reply:
ok